csharp/src/Google.Protobuf/project.json

```diff
@@ -0,0 +1,10 @@
+﻿{
```

Comment:
That's only if we're using all the dotnet CLI tooling - which isn't the plan just yet. The project.json is *additional* to the csproj file, and it should be fine to use the existing nuspec file.

I should probably explain that most of this PR was created by going into project properties in VS and hitting "Target .NET Standard Library"... VS made the changes to the csproj file and created the project.json file.